controllers/grid/files/proof/ProofFilesGridHandler.inc.php

```diff
@@ -42,6 +46,19 @@ function ProofFilesGridHandler() {
 	// Implement template methods from PKPHandler
 	//
 	/**
+	 * @copydoc PKPHandler::authorize()
+	 */
+	function authorize($request, &$args, $roleAssignments) {
+		import('classes.security.authorization.SubmissionAccessPolicy');
+		$this->addPolicy(new SubmissionAccessPolicy($request, $args, $roleAssignments));
```

Comment:
the parent class also already implements the WorkflowStageAccessPolicy, which uses the submission access policy already. So no need for authorize method here, I think.